parlai/scripts/detect_offensive_language.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) 2017-present, Facebook, Inc.
+# All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+"""Basic example which iterates through the tasks specified and
+checks them for offensive language.
+
+For example:
+`python parlai/scripts/detect_offensive_language.py -t "convai_chitchat" --display-examples True -dt train`
+"""
+from parlai.core.params import ParlaiParser
+from parlai.core.agents import create_agent
```

Comment:
don't need to import this

Reply:
i updated so you can actually put a model through it as well